webhooks-extension/config/latest/openshift-tekton-webhooks-extension.yaml

```diff
@@ -296,19 +296,19 @@ spec:
     image: maiwj/kubernetes-python-client:latest
```

Comment:
Is this another folder name change? If so please check what's at https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/InstallNightlyBuild.md  and anything else at https://github.com/tektoncd/experimental/tree/master/webhooks-extension/docs